extractor/contents.py

```diff
@@ -0,0 +1,151 @@
+import enum
+
+class Contents:
+    def __init__(self):
+        self.content_list = list()
+
+    def add_content(self, _content):
+        self.content_list.append(_content)
+
+    def __str__(self):
+
+        return_string = '{\n'
+
+        for content in self.content_list:
+            content_string = '\t{Index: ' + str(content.index) + '\n'
+            content_string += '\t\tContentType: ' + content.contentType.name + '\n'
+
+            if content.contentType == ContentType.TEXT:
+                content_string += '\t\t' + str(content.text)
+            elif content.contentType == ContentType.IMAGE:
+                content_string += '\t\t' + str(content.image)
+            elif content.contentType == ContentType.VIDEO:
+                content_string += '\t\t' + str(content.video)
+            elif content.contentType == ContentType.QUOTE:
+                content_string += '\t\t' + str(content.quote)
+            elif content.contentType == ContentType.EMBEDDED_TWEET:
+                content_string += '\t\t' + str(content.tweet)
```

Comment:
You can use [string format](https://docs.python.org/3.4/library/string.html#string-formatting) to avoid append values to string multiple times.

Reply:
It can apply to every string in this file.

Reply:
Updated accordingly! Thanks for suggestion.